src/model/itemUrls.ts

```diff
@@ -11,12 +11,18 @@
  */
 
 export class ItemUrls {
+    'label'?: string;
```

Comment:
I assume the "?" here means optional - a glance at the [spec](https://developer.1password.com/docs/connect/connect-api-reference/#item-url-object) it wasn't clear to me whether it was or not.

Reply:
Yeah, it's optional. There is an opportunity to create an api without a label and in this case the api returns `undefined`.

I'll add an additional test case.